apps/image-server/server.js

```diff
@@ -111,22 +114,6 @@ const argv = require('yargs')
   .help()
   .argv;
 
-passport.use(new Strategy(
```

Comment:
Op zich wel mooi als dit er uit gaat: dan heeft de image server helemaal geen database meer nodig. Kun je dat er dan ook uit slopen, of een ticket maken dat dat moet gebeuren.

Reply:
Deze heb ik op het trello bord gezet voor na dit alles!